Source/WebCore/loader/ResourceMonitor.cpp

```diff
@@ -64,6 +56,7 @@ void ResourceMonitor::setEligibility(Eligibility eligibility)
         return;
 
     m_eligibility = eligibility;
+    RESOURCEMONITOR_RELEASE_LOG("The frame is%s eligible.", (eligibility != Eligibility::Eligible ? " not" : ""));
```

Comment:
nit: missing a space before `%s`.

I don't remember the exact rules for `os_log` (which is what this compiles down to). In theory, a string with static storage duration should be emitted when using a bare `%s`. But I'm not sure if this works if the string literal is the result of a ternary expression because the way that work might be via something like `__builtin_constant_p(x)`.

You probably want to explicitly use `%" PUBLIC_LOG_STRING` instead to be sure (which compiles down to `%{public}s`:

```
RESOURCEMONITOR_RELEASE_LOG("The frame is %" PUBLIC_LOG_STRING " eligible.", (eligibility != Eligibility::Eligible ? " not" : ""));
```